packages/config/package.json

```diff
@@ -5,12 +5,11 @@
   "main": "build/src/index.js",
   "types": "build/src/index.d.ts",
   "scripts": {
+    "build": "tsc",
     "format": "prettier --check .",
     "format:fix": "prettier --write .",
     "lint": "eslint --ext .ts --max-warnings 0 src test",
     "lint:fix": "yarn lint --fix",
-    "typecheck": "tsc --noEmit",
```

Comment:
Why are we removing typecheck commands?

Reply:
Because it's redundant. Build already covers that use case, but only for some packages. Removing that command skips running it for those packages on CI